build/steps-build.yml

```diff
@@ -112,6 +112,15 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: "$(Build.SourcesDirectory)/src/coverage.cobertura.xml"
 
+- task: nventiveReleaseNotesCompiler@5
```

Comment:
Is there doc for this task somewhere?

Reply:
Not really no, I can make something in the repository! It's really basic, that'll be quick

Reply:
I think comments right there would be fine